### PR TITLE
Add wdltools

### PR DESCRIPTION
**Pull request by rhpvorderman**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/wdltools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dnanexus-rnd/wdlTools/archive/{{ version }}.tar.gz
+  sha256: 115fd4ce8ae76e2963b284617bc14c72f494b11a4bef51c67baabd95d2a01974
+
+build:
+  skip: true  # [not unix]
+  number: 0
```
Comment (scopatz) [inline recipes/wdltools/meta.yaml:14]: Can this be `noarch: generic`? CC @mariusvniekerk

Comment (mariusvniekerk) [reply recipes/wdltools/meta.yaml:14]: Given the structure of the entrypoint no.

**Comment by jdidion** [comment]
I am a maintainer - I am willing to be listed.

**Comment by rhpvorderman** [comment]
@conda-forge/help-java This package is ready to be reviewed and merged.

@jdidion I have removed Ohad as a maintainer so the package can be merged. After this is merged it will get its own feedstock where you have write access. It should be very easy to add Ohad afterwards if he also wants maintainer access to the package.

**Comment by rhpvorderman** [comment]
@jdidion yes, and cromwell needs 8 as well. So I thought it would be nice to be able to install cromwell and wdltools in the same environment. (Together with miniwdl, pytest-workflow, wdl-aid, wdl-packager etc.). Come to think of it, we should add pytest-wdl to conda as well. It is quite straightforward to add new python packages. Using `conda skeleton pypi pytest-wdl` should already generate a working meta.yml file.

**Comment by rhpvorderman** [comment]
@conda-forge/help-java This package is ready to be reviewed and merged.

**Comment by rhpvorderman** [comment]
@conda-forge/staged-recipes Unfortunately the java team is unresponsive. 
Can this recipe be merged?

File: recipes/wdltools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+requirements:
+  build:
+    - openjdk =8
+    - sbt
+    - make
+    - wget
+  run:
+    - openjdk =8|11
```
Comment (scopatz) [inline recipes/wdltools/meta.yaml:23]: Is this a valid spec?  would `!9,!10` work?

Comment (rhpvorderman) [reply recipes/wdltools/meta.yaml:23]: Yes it is.
8 and 11 are LTS releases, so they are explicitly asked for here.

**Comment by scopatz** [comment]
Thanks!

**Comment by rhpvorderman** [comment]
@scopatz Thank you!

**Pull request merged**